plugins/ja3_fingerprint/ja3_fingerprint.cc

```diff
@@ -431,9 +442,9 @@ TSRemapDoRemap(void *ih, TSHttpTxn rh, TSRemapRequestInfo *rri)
 void
 TSRemapDeleteInstance(void *ih)
 {
-  auto pri = static_cast<ja3_remap_info *>(ih);
-  if (pri) {
-    delete pri;
+  auto remap_info = static_cast<ja3_remap_info *>(ih);
+  if (remap_info) {
```

Comment:
I don't think you need this - `delete` should handle `nullptr`.

Reply:
Good point. I removed the if-guard.